packages/data-context/test/unit/actions/project/counter-multiple-components.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export function CounterContainer () {
```

Comment:
I would like to see some different syntaxes used just to make sure our babel parsing/libraries can handle them:

1. const arrow function:
```
const Counter = () => <p>Counter</p>
```
2. Separate export statements:
```
function Counter () { return <p>Counter</p> }
function OtherCounter() { return <p>OtherCounter</p> }

export {
  Counter
}
export default OtherCounter
```